Documentation/Manual/Texinfo/chap-inspector.texi

```diff
@@ -32,87 +42,336 @@ To get up and running quickly with Clouseau:
 
 @enumerate
 @item
- With ASDF and @mcclim loaded, load the file
+Load the @t{clouseau} system with @cl{(ql:quickload
+"clouseau")}.  Alternatively, use @cl{(asdf:load-system "clouseau")},
+potentially after manually loading
+@file{mcclim/Apps/Clouseau/clouseau.asd}.
 
-  @file{mcclim/Apps/Clouseau/clouseau.asd}.
 @item
- Load Clouseau with:
+Inspect an object with @cl{(@funref{clouseau:inspect} @var{object})} where
+@var{object} can be any Lisp object.  If you use a multithreaded Lisp
+implementation, you can also include the @cl{:new-process} keyword
+argument.  If it is true, then Clouseau is started in a separate thread,
+causing the above call to return immediately.
+
+For example, executing the following code
+
+@lisp
+(defclass foo () ((a :initarg :a) (b :initform #(1 2 3)) (c :initarg :c)))
+(@funref{clouseau:inspect} (make-instance 'foo :c (make-instance 'foo)))
```

Comment:
Do not use refrences in the listing. It can be tolerated in HTML but completely messes the pdf and Info output. The refrence is already in the description above.

Reply:
I would like to keep the references for HTML, but you are of course totally right, that the other outputs must not be negatively affected.

I have a solution based on defining the macros differently for HTML output (https://github.com/scymtym/McCLIM/commit/809949d403a5f66ed420e9267e255bd1f6213fb2). Would this be acceptable for your?

(I also reformatted some of the listings to better fit into the width available in the PDF output.)

Reply:
Your solution is better.

Reply:
Ok, I made #813 to add the infrastructure.